exporter/elasticsearchexporter/model.go

```diff
@@ -985,7 +986,11 @@ func valueHash(h hash.Hash, v pcommon.Value) {
 		h.Write(buf)
 	case pcommon.ValueTypeInt:
 		buf := make([]byte, 8)
-		binary.LittleEndian.PutUint64(buf, uint64(v.Int()))
+		var i uint64
+		if v.Int() > 0 {
+			i = uint64(v.Int()) //nolint:gosec // overflow checked
+		}
+		binary.LittleEndian.PutUint64(buf, i)
```

Comment:
we need a valid hash for negative values as well.

Reply:
For this case, maybe relying on the value higher than `MaxInt64` that Go will use is better?

Reply:
Yes, it is just fine as long as any number in int64 is mapped into a []byte with unique content.